TimeService/TimeService.asn

```diff
@@ -0,0 +1,12 @@
+TIMESERVICE-DATAVIEW DEFINITIONS ::=
+BEGIN
+   UInteger ::= INTEGER (0 .. 4294967295)
+   ULongInteger ::= INTEGER (0 .. 18446744073709551615)
+   ComparisonResult ::= INTEGER (-1 .. 1)
+   ClockStatusEnum  ::= ENUMERATED { unavailable, available }
+   CucTimestamp ::= OCTET STRING (SIZE(8))
```

Comment:
Didn't we discuss making it 7 bytes?

Reply:
We did I think I accidentally did not include the change here

Reply:
Fixed